Makefile-az.mk

```diff
@@ -25,6 +25,32 @@ az-all-custom-vnet: az-login az-create-workload-msi az-mkaks-custom-vnet az-crea
 
 az-all-savm:    az-login az-mkaks-savm az-perm-savm az-patch-skaffold-azure az-build az-run az-run-sample ## Provision the infra (ACR,AKS); build and deploy Karpenter; deploy sample Provisioner and workload - StandaloneVirtualMachines
 
+install-tools: az-tool ## install tools to create a local developer environment
+	sudo apt update && sudo apt install jq -y
+	curl -Lo yq https://github.com/mikefarah/yq/releases/download/v4.43.1/yq_linux_amd64 && chmod +x yq && sudo mv yq /usr/local/bin
+	if test -z "$(shell which helm)"; then \
+		curl -L https://get.helm.sh/helm-v3.14.3-linux-amd64.tar.gz | sudo tar xvfz - linux-amd64/helm && sudo mv linux-amd64/helm /usr/local/bin/helm && rm -rf linux-amd64; \
```

Comment:
Instead of curling `get.helm.sh` I'm seeing the following recommendation from the helm site?:
https://helm.sh/docs/intro/install/
![image](https://github.com/Azure/karpenter-provider-azure/assets/33269602/dcf25981-a80f-4d00-8cb8-5304034506a2)

If there is a reason to adjust it though, that makes sense. Mostly curious.

Reply:
Whenever there is an alternative, let's avoid piping something into bash, just a good security practice. (Yes, this probably means paying the price of managing versions explicitly ...)